drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3752,6 +3787,10 @@ RasterizerSceneGLES3::~RasterizerSceneGLES3() {
 	RSG::material_storage->material_free(scene_globals.default_material);
 	RSG::material_storage->shader_free(scene_globals.default_shader);
 
+	// Overdraw Shader
+	RSG::material_storage->shader_free(scene_globals.overdraw_material);
```

Comment:
```suggestion
	RSG::material_storage->material_free(scene_globals.overdraw_material);
```

Reply:
wow, seems like nothing slips by you and these automated checks